intercom_flutter/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 7.6.8
```

Comment:
If this wil require a change to all Android users, I think this version should be `8.0.0` instead. Also, if Android initialization will be done in the native side now, can we remove the `androidApiKey` from the Dart `initialize` method?

Reply:
It's not a breaking change.

Reply:
Sorry to interrupt here but this changes will prevent the app from crashing